<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReathaDesktop</class>
 <widget class="QMainWindow" name="ReathaDesktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReathaDesktop</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeView" name="tvPrjView">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="autoExpandDelay">
       <number>-1</number>
      </property>
      <property name="indentation">
       <number>19</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="swEditViews">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="wpProject">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Edit Project </string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>API URL</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="eProjectUrl">
               <property name="text">
                <string>http://reatha.de/rm/api/</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>User</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="eProjectUser"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="eProjectPasswd">
               <property name="echoMode">
                <enum>QLineEdit::PasswordEchoOnEdit</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Location</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="text">
                <string>c:/temp/</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bSelectProjectDir">
               <property name="text">
                <string>Select Dir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="bDownloadProject">
               <property name="text">
                <string>Download</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bUploadProject">
               <property name="text">
                <string>Upload</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>400</height>
           </size>
          </property>
          <property name="title">
           <string>Log</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextBrowser" name="eLog"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wpCommon"/>
      <widget class="QWidget" name="wpDomain"/>
      <widget class="QWidget" name="wpDevice"/>
      <widget class="QWidget" name="wpView"/>
      <widget class="QWidget" name="wpDomainAdmin"/>
      <widget class="QWidget" name="wpUser"/>
      <widget class="QWidget" name="wpVariable"/>
      <widget class="QWidget" name="wpTransformation"/>
      <widget class="QWidget" name="wpNotificationRule"/>
      <widget class="QWidget" name="wpUserNotification"/>
      <widget class="QWidget" name="wpImage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionNew_project"/>
    <addaction name="actionNew_Domain"/>
    <addaction name="actionNew_Domain_Admin"/>
    <addaction name="actionNew_Device"/>
    <addaction name="actionNew_View"/>
    <addaction name="actionNew_User"/>
    <addaction name="actionNew_Variable"/>
    <addaction name="actionNew_Image"/>
    <addaction name="actionNew_Notification_Rule"/>
    <addaction name="actionDownload_Project"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommands"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_project">
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionNew_Domain">
   <property name="text">
    <string>New Domain</string>
   </property>
  </action>
  <action name="actionNew_Device">
   <property name="text">
    <string>New Device</string>
   </property>
  </action>
  <action name="actionNew_View">
   <property name="text">
    <string>New View</string>
   </property>
  </action>
  <action name="actionNew_Domain_Admin">
   <property name="text">
    <string>New Domain Admin</string>
   </property>
  </action>
  <action name="actionNew_User">
   <property name="text">
    <string>New User</string>
   </property>
  </action>
  <action name="actionNew_Variable">
   <property name="text">
    <string>New Variable</string>
   </property>
  </action>
  <action name="actionNew_Image">
   <property name="text">
    <string>New Image</string>
   </property>
  </action>
  <action name="actionNew_Notification_Rule">
   <property name="text">
    <string>New Notification Rule</string>
   </property>
  </action>
  <action name="actionDownload_Project">
   <property name="text">
    <string>Download Project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
